core/src/storage2/collections/binary_heap/elements.rs

```diff
@@ -12,13 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Wrapper which provides an interface around the vector used to store
-//! elements of the [`BinaryHeap`](`super::BinaryHeap`) in storage.
+//! Provides an interface around the vector used to store elements of the
+//! [`BinaryHeap`](`super::BinaryHeap`) in storage. This is necessary since
+//! we don't just store each element in it's own storage cell, but rather
+//! optimize storage access by putting children together in one storage cell.
 
 use crate::storage2::{
     collections::binary_heap::{
```

Comment:
This is the same module so could be `super`

Reply:
also both PRs (this and its base PR) must be rebased on `master` before we can merge.